### PR TITLE
Integration/0.10.0.1

### PR DESCRIPTION
**Pull request by chrisguida**

### CODE REVIEW

File: assets/compat/btc-rpc-proxy.rules.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "getnetworkinfo"
+- rule: 'users.[first(item => ''item.name = "c-lightning")].fetch-blocks?'
+  description: 'RPC user "c-lightning" must have "Fetch Blocks" enabled'
+  suggestions:
+    - SET:
+        var: 'users.[first(item => ''item.name = "c-lightning")].fetch-blocks'
+        to-value: true
```
Comment (elvece) [inline assets/compat/btc-rpc-proxy.rules.yaml:67]: has it been verified these attributes are actually set in proxy?

**Review by elvece — APPROVED** [review]

File: assets/compat/btc-rpc-proxy.rules.yaml (hunk 1 above)
Comment (chrisguida) [reply assets/compat/btc-rpc-proxy.rules.yaml:67]: If you mean rpc commands in general, yes they are working as expected. If you mean "fetch-blocks", no, I haven't tested that one specifically.

**Comment by chrisguida** [comment]
I think c-lightning is ready to go, except we need to update to 0.10.2 for security update. Will try working around by building on debian instead of alpine https://github.com/ElementsProject/lightning/pull/4950

**Comment by chrisguida** [comment]
Closing in favor of https://github.com/Start9Labs/c-lightning-wrapper/pull/20

**Pull request closed without merging**